conda.recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ requirements:
     - python
     - click
     - h5py
-    - nrel-pysam==1.2.1
+    - nrel-pysam>=2.1.1
```

Comment:
@MRossol, im assuming this is all I need to do to update the Conda env for pysam2?

Reply:
yup!